### DISCUSSION TITLE
Remove Ethernet

### DISCUSSION THREAD

**Discussion by jbpoint** [vmware/packer-examples-for-vsphere#149] [Help] [answered]
Hello,

is there a way to remove the network adapter while the template has been created? 
Because i set a static IP as long as the template builds and after that i want to remove the whole adapter that when i boot the VM there isnt a network configured

**Comment by tenthirtyam** [comment, marked as answer]
This use case is not covered by this project, nor is the functionality to remove the adapter(s) available in the vSphere [post-processors](https://www.packer.io/docs/post-processors) for Packer.

If you are performing guest customization via native vSphere or invoking customization via Terraform the network settings used during the build should be removed.

Ryan
